src/olympia/lib/settings_base.py

```diff
@@ -261,7 +261,7 @@ def lazy_langs(languages):
 
 # paths that don't require an app prefix
 # This needs to be kept in sync with addons-frontend's validUrlExceptions
-# https://github.com/mozilla/addons-frontend/blob/master/config/default-amo.js#L27
+# https://github.com/mozilla/addons-frontend/blob/master/config/default-amo.js#L30
```

Comment:
Now that I'm looking at it, I'm not a fan of referencing the actual lines for exactly this reason. There are worse things though, so ...